doc/source/amazon-rds.rst

```diff
@@ -92,13 +92,13 @@ To set up OS metrics monitoring for Amazon RDS in PMM via CloudWatch:
 
    .. code-block:: bash
 
-      $ docker run -d \
+      [root@pmm-server ~]# docker run -d \
         -p 80:80 \
         --volumes-from pmm-data \
         -v /path/to/file/with/creds:/usr/share/grafana/.aws/credentials \
         --name pmm-server \
         --restart always \
-        percona/pmm-server:1.2.0
+        percona/pmm-server:latest
```

Comment:
@michaelcoburn I did not, but it was tested by Roman Vynar when he initially gave me the instructions and then confirmed that they work as described.